components/datetime/tests/fixtures/structs.rs

```diff
@@ -1,45 +1,33 @@
 // This file is part of ICU4X. For terms of use, please see the file
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/master/LICENSE ).
+#![cfg(not(feature = "serialize_none"))]
+
+use icu_datetime::options::{components, style};
 use serde::{Deserialize, Serialize};
 
-#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
```

Comment:
I removed the `Clone` and `PartialEq` traits, as these propagate down to every type in the serialized structure.

Reply:
This comment is outdated now.